packages/cmrjs/utils.js

```diff
@@ -187,6 +114,8 @@ async function validateUMMG(ummMetadata, identifier, provider) {
  * static string if not defined. The value returned should be a valid IP address or
  * else the request for a CMR token will fail.
  *
+ * @deprecated
```

Comment:
If these are deprecated, we should use the `deprecate()` function, like above and specify the version they are deprecated in.

Reply:
Make sure to use a copy of the `deprecate()` function, don't use `@cumulus/common/util#deprecate()`, or else you're going to import a ton of node-specific stuff (and the `aws-sdk` library), which was what we were trying to avoid with the change.

Reply:
Yep, good point. Thanks @yjpa7145. I'm actually even ok if we don't use the function, I just need to know the version for when I do the cleanup, although it's nice to have it tell the user the function is deprecated.

Reply:
isn't that what's being pulled in here? https://github.com/nasa/cumulus/pull/842/files#diff-db1edf76b193b226fefdb1308e0c96ccR4 (deprecate from `@cumulus/common/util`?)

Reply:
You're correct.  When I made my comment, I was thinking that this was in the `cmr-client` package, which does not (and should not) have a dependency on `@cumulus/common`.  We can definitely use `@cumulus/common/util.deprecate()` here.